soil/web.py

```diff
@@ -210,9 +211,12 @@ def ParseJobs(stdin):
     &nbsp;
     <code><a href="https://github.com/oilshell/oil/commit/%(commit-hash)s">%(commit_hash_short)s</a></code>
   </td>
-  <td class="commit-line" colspan=4>
+  <td class="commit-line" colspan=3>
     <code>%(commit-line)s</code>
   </td>
+  <td colspan=1>
+    <code><a href="https://github.com/oilshell/oil/pull/%(pr-number)s">#%(pr-number)s</a></code>
```

Comment:
This will display a blank `#` when there's no PR

But OK let's just run it and see what it looks like

We need a template language with conditionals ...